Classes/FrontEnd/DefaultController.php

```diff
@@ -2586,13 +2586,4 @@ protected function getConfigurationWithFlexForms(): Configuration
 
         return $this->configuration;
     }
-
-    private function renderAsRichText(string $rawData): string
-    {
-        $arguments = ['parseFuncTSPath' => 'lib.parseFunc_RTE'];
-        $childrenClosure = function () use ($rawData): string {
-            return \trim($rawData);
-        };
-        return HtmlViewHelper::renderStatic($arguments, $childrenClosure, new NullRenderingContext());
```

Comment:
We probably then also can remove the `use` of `HtmlViewHelper` and `NullRenderingContext` now.